craft_providers/lxd/lxd_instance.py

```diff
@@ -86,6 +64,42 @@ def __init__(
         else:
             self.lxc = lxc
 
+    def _rootify_lxc_command(
+        self,
+        command: List[str],
+        *,
+        env: Optional[Dict[str, Optional[str]]] = None,
+    ) -> List[str]:
+        """Wrap a command to run as root with specified environment.
+
+        Formulate command to:
+
+            - Use sudo to run as root (LXD defaults to root, but doesn't
+              initialize environment like sudo will, e.g. read
+              /etc/environment).  For consistency with Multipass and other
+              providers, always use sudo.
+
+            - Configure sudo to set home directory.
```

Comment:
I'm a bit confused by this comment, I don't see this happening in the implementation. Is it being done elsewhere?

Reply:
Ah, that's outdated, will fix.  Up until recently it used sudo.

Reply:
Updated the docs and renamed the method too.